schema/mysql-upgrades/1.0.0.sql

```diff
@@ -13,6 +15,13 @@ BEGIN
   DECLARE schedule CURSOR FOR SELECT id, start, frequency, config FROM schedule;
   DECLARE CONTINUE HANDLER FOR NOT FOUND SET done = 1;
 
+  -- Determine the current session time zone name
+  SELECT IF(@@session.TIME_ZONE = 'SYSTEM', @@system_time_zone, @@session.TIME_ZONE) INTO session_time_zone;
+
+  IF (session_time_zone NOT LIKE '+%:%' OR session_time_zone NOT LIKE '-%:%') AND CONVERT_TZ(FROM_UNIXTIME(1699903042), session_time_zone, '+00:00') IS NULL THEN
```

Comment:
If it's *like* +%:%, it's *unlike* -%:%
If it's *like* -%:%, it's *unlike* +%:%

You want *if unlike +%:% and unlike -%:% and convert_tz(...) is null*, don't you?